frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/Platform.java

```diff
@@ -66,14 +66,15 @@ public String getLongNodeFilename(String nodeVersion) {
 
     public String getNodeDownloadFilename(String nodeVersion) {
         if(isWindows()) {
+			int versionCompare = nodeVersion.compareTo("v4.0.0");
             if(architecture == Architecture.x64){
-                if (nodeVersion == "v4.0.0") {
+				if (versionCompare >= 0) {
```

Comment:
I thought this was just fixing the wrong comparison [of the last PR](https://github.com/eirslett/frontend-maven-plugin/commit/fa6430d9bbf15b4d3a99a3ca63beda900b54c03d). Sorry 'bout that.